src/views/reset.ejs

```diff
@@ -32,36 +32,38 @@
                                 <%= alert.message %>
                             </div>
                             <% } %>
-                        <% if (alert && alert.variant == 'success') { %>
-                        <div class="alert alert-<%= alert.variant %>">
-                            <span>Your password has been changed Successfully!<br />
-                                <a class="btn btn-link px-0 py-1" href="/interaction/<%= uid %>/abort">Go to Sign in.</a>
```

Comment:
@JustinaMary Replaced this abort link with the absolute return_url, since going back to the return_url will request a login and aborting resulted in an error for me.

Reply:
Yeah got you